test/conformance/parallel/consumer-sidecar.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cloud-proxy-sidecar
+  namespace: openshift-ptp
+  labels:
+    app.kubernetes.io/name: proxy
+spec:
+  containers:
+  - name: sidecar
+    image: quay.io/redhat-cne/cloud-event-proxy
```

Comment:
Please use image form ptp-operator . rehdat-cne image may not match with version that is deployed by ptp operator

Reply:
fixed